tests/contract-playground/src/ArithmeticUnderflowOverflow.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: MIT
+pragma solidity <0.8.0;
+ 
+contract ArithmeticUnderflowOverflow {
+    uint256 public counter;
+
+    // bad
```

Comment:
I've added a lot more test cases to this file. The functions commented `// bad` should be caught by this detector, and the ones commented `// good` should not be.